jest.config.ts

```diff
@@ -119,12 +119,18 @@ export default {
       testEnvironment: 'node',
       preset: 'ts-jest',
       testMatch: ['<rootDir>/packages/server/test/**/*.spec.ts'],
+      moduleNameMapper: {
+        '@openfeature/core': '<rootDir>/packages/shared/src'
+      },
```

Comment:
This took a while to figure out... because we now re-export the shared module, depending on the import path that was resolved, sometimes we'd resovel `../shared/src` in tests and sometimes we'd resolve `../shared/dist` (whatever was last built). This caused all kinds of weird test failures.

This module rewrite forces Jest to always resolve the source files, not what's sitting in the `@openfeature/core dist/` folder.